pandas/tests/io/test_excel.py

```diff
@@ -26,13 +26,21 @@
 from pandas.io.formats.excel import ExcelFormatter
 from pandas.io.parsers import read_csv
 
-_seriesd = tm.getSeriesData()
-_tsd = tm.getTimeSeriesData()
-_frame = DataFrame(_seriesd)[:10]
-_frame2 = DataFrame(_seriesd, columns=['D', 'C', 'B', 'A'])[:10]
-_tsframe = tm.makeTimeDataFrame()[:5]
-_mixed_frame = _frame.copy()
-_mixed_frame['foo'] = 'bar'
+
+@pytest.fixture
+def frame(float_frame):
+    return float_frame[:10]
+
+
+@pytest.fixture
+def frame2(float_frame):
+    float_frame.columns = ['D', 'C', 'B', 'A']
```

Comment:
there's a couple of tests where frame and frame2 are both injected into the same test. i see that in one case it's to write to multiple sheets. can you check that all tests with frame and frame2 together need them both otherwise the tests should be split.

Reply:
oh. and you should do a copy of float frame before changing the column labels because of that.

Reply:
tbh I think we can get rid of this fixture altogether. It's only used in two tests and one of them ignores the header changes anyway so no real point in having this

Reply:
For now to minimize diff going to add copy as you suggested, but removing should make for an easy follow up PR

Reply:
yeah, probably best not to change the test behavior here, but if it's not adding anything then can remove in a follow-on.